code/modules/admin/holder2.dm

```diff
@@ -48,8 +56,20 @@ GLOBAL_PROTECT(href_token)
 	else
 		deactivate()
 
+/datum/admins/proc/Destroy()

```

Comment:
you's a chumbis

Reply:
kek, i'll fix it in a bit, gotta do the same for ranks